packages/astro/package.json

```diff
@@ -77,7 +77,8 @@
       "types": "./dist/core/middleware/namespace.d.ts",
       "default": "./dist/core/middleware/namespace.js"
     },
-    "./transitions": "./dist/transitions/index.js"
+    "./transitions": "./dist/transitions/index.js",
+    "./i18n": "./dist/i18n/index.js"
```

Comment:
Since we have had to do this so many times for our namespaced modules, maybe we should have a convention for this to avoid repeating the exports, like `"./modules/*": "./dist/modules/*"`. Then anything we put in that folder would be exposed. Not something to change in this PR, just a thought for a future refactor.

Reply:
I'd would to defer this change to someone else, preferably in another PR.